Libraries/DirectMLX.h

```diff
@@ -4291,6 +4291,191 @@ namespace dml
 
         return output;
     }
+
+    struct MultiHeadAttentionOutputs
+    {
+        Expression output;
+        Optional<Expression> outputPresentKey;
+        Optional<Expression> outputPresentValue;
+    };
+
+    inline MultiHeadAttentionOutputs MultiHeadAttention(
```

Comment:
```
    inline MultiheadAttentionOutputs MultiheadAttention(
```
Multihead is a single word (https://en.wiktionary.org/wiki/multihead / https://www.merriam-webster.com/dictionary/multiheaded), consistent with our enum `DML_MULTIHEAD_ATTENTION_OPERATOR_DESC` and with PyTorch (https://pytorch.org/docs/stable/generated/torch.nn.MultiheadAttention.html).